NEWS.rst

```diff
@@ -1,3 +1,8 @@
+Features
+--------
+- Removed Great Prayer Day from Denmark holidays from year 2024.
+
+
```

Comment:
I know the PR template says to put the changelog here... but that's an artifact of workalendar. I need to update that. In the meantime, I'll move this entry to a towncrier news fragment.